unic/ucd/name/src/name.rs

```diff
@@ -148,4 +207,5 @@ mod data {
     use unic_char_property::tables::CharDataTable;
     include!("../tables/name_values.rsd");
     pub const NAMES: CharDataTable<&[&str]> = include!("../tables/name_map.rsv");
+    pub const JAMO_SHORT_NAMES: CharDataTable<&str> = include!("../tables/jamo.rsv");
```

Comment:
Considering the fact that these values are not that useful directly, how about we just make both of these private?

Reply:
I tried to remove the `pub`, but looks like it will make these constants non-accessible from `Name` :(

Reply:
Oh, these are only `pub` in the private `mod data`. So, yeah, need to be `pub` and is not exposed to the outside of the crate, meaning it's fine. Thanks for looking!